guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -406,10 +517,12 @@
         "FIELD_HEADER_TYPESCRIPT_PATH" : "Ruta Typescript :",
         "FIELD_HEADER_USE_SSL"         : "Usa SSL/TLS",
 
+        "FIELD_OPTION_BACKSPACE_EMPTY" : "",
         "FIELD_OPTION_BACKSPACE_8"     : "Espai enrere (Ctrl-H)",
         "FIELD_OPTION_BACKSPACE_127"   : "Eliminar (Ctrl-?)",
 
         "FIELD_OPTION_COLOR_SCHEME_BLACK_WHITE" : "Negre sobre blanc",
+        "FIELD_OPTION_COLOR_SCHEME_EMPTY"       : "",
```

Comment:
No need to include this, as it requires no translation.

Reply:
Removed